### ISSUE TITLE
Add ChatApp

### ISSUE THREAD

**Issue by mishal23** [mishal23/web-projects#7]

**Comment by drashti4** [comment]
Are you considering bot inside the chat application?

**Comment by mishal23** [comment]
No @drashti4 , any kind of chat application is fine!
Bots one is also fine, normal between two users is also fine

**Comment by drashti4** [comment]
cool! Any specific format for it? Cause I couln't find chatApp folder

**Comment by mishal23** [comment]
Yes make a new folder and add a ```README.md``` in it, and the same format as mentioned in samples folder
